test/extended/testdata/deployments/deployment-simple.yaml

```diff
@@ -3,7 +3,7 @@ kind: DeploymentConfig
 metadata:
   name: deployment-simple
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
e.g. we test #availableReplicas that is dependent on #replicas, but I have fixed some tests to actually read the number of replicas from the fixture

Reply:
sure, lets wait for the tests to finish... i don't think we need 2 replicas here because it means twice more time to wait for the pod to be ready and twice more chance something gets wrong with the second replica.

Reply:
I am fine with 1 by default. Any test that would require more is free to overwrite the value explicitly. (There are special cases where having more replicas leads to more stability - like when you want to avoid detecting false detach when the pod dies.)

Reply:
seems like it was just one test :) I was expecting more :)